maple-sdk/protocol-actors.md

```diff
@@ -0,0 +1,98 @@
+# Protocol Actors
+
+There are three actors in the Maple Protocol ecosystem:
+
+- Borrowers
+- Lenders
+- Pool Delegates
+
+# Borrowers
+
+- Borrowers request loans with specific terms and can provide collateral as required.
+- Borrowers can drawdown loans once approved and funded.
+- Borrowers are responsible for repaying the principal and interest payments through the same smart contracts.
+
+**Code Example:**
+This snippet queries the `GlobalsV2` contract to check if a given address is an authorised borrower
```

Comment:
Good example here where a better explanation about globals would help give this more context

Also can we check with Charlotte and Graeme on our English style? IIRC we were going with z not s for words like authorized. I dont have a strong opinion but given this is outward facing want to ensure Maple is speaking with a consistent voice across all areas

Reply:
I'm wary of re-documenting the smart contracts so I'll link `globals` to the main SC docs